### PR TITLE
Adding missing include guards.

### PR DESCRIPTION
**Pull request by scottdarch**

### CODE REVIEW

File: Adafruit_ADXL343.h [hunk 1]
```diff
@@ -163,3 +166,5 @@ class Adafruit_ADXL343 : public Adafruit_Sensor {
   uint8_t _clk, _do, _di, _cs;
   bool    _i2c;
 };
+
+#endif # _ADAFRUIT_SENSOR_ADXL343_H
```
Comment (ladyada) [inline Adafruit_ADXL343.h]: remove extra #

**Review by ladyada — CHANGES_REQUESTED** [review]

File: Adafruit_ADXL343.h (hunk 1 above)
Comment (scottdarch) [reply Adafruit_ADXL343.h]: Oh weird. It's merging from my personal account but says that my work account (@thirtytwobits) is doing the comments. For the record (geesh, lawyers. Am I right :) this change is submitted by Scott Dixon via his scottdarch account and is in no way associated with Amazon.com nor the UAVCAN open source project. I claim no copyrights for this change.

Comment (ladyada) [reply Adafruit_ADXL343.h]: ha ok do you just wanna close this PR and make a new one?

Comment (scottdarch) [reply Adafruit_ADXL343.h]: No. I think it's cool. The change comes from my personal account. That's all that matters.

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
thanks :)

**Pull request merged**